actionManifest-webcraft.json

```diff
@@ -0,0 +1,62 @@
+{
+    "tag": "quiz-teams-app",
```

Comment:
this is only for development purpose right ?

Reply:
Yes, Removed now